phoenix-core-client/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -755,6 +758,18 @@ public Pair<Integer, Tuple> call() throws SQLException {
         }
     }
 
+    private static boolean isSingleRowUpdate(boolean isUpsert, boolean isDelete,
+                                             MutationPlan plan) {
+        boolean isSingleRowUpdate = false;
+        if (isUpsert) {
+            isSingleRowUpdate = true;
+        } else if (isDelete) {
+            isSingleRowUpdate =
+                    plan.getContext().getScanRanges().isPointLookup();
```

Comment:
What about multi key point lookup ? It is possible for delete also using IN clause.

Reply:
Even then we won't return any result because `MutationState` changes will ensure that only if we are creating PRow for Delete, we use result return attribute. Let me add a test to ensure multi point lookup with SkipScan filter is covered.

Reply:
Why will Delete use a SkipScan ?

Reply:
I am talking about the case where multiple full row keys are provided. There shouldn't be any SkipScan.

Reply:
Covered in the test, it still uses PRow for Delete only if single row is deleted with point lookup plan rather than multi key point lookup.

Reply:
So in this case all the Delete mutations sent by the client won't have the return result attribute ?

Reply:
That is correct. Only if single key point lookup is used with auto-commit and the new JDBC API, only then the attribute will be sent.